ui/pages/permissions-connect/snaps/snap-install/snap-install.js

```diff
@@ -179,6 +179,7 @@ export default function SnapInstall({
           onCancel={() => setIsShowingWarning(false)}
           onSubmit={onSubmit}
           warnings={warnings}
+          snapName={snapName}
```

Comment:
Please update this on `snap-update` too!

Reply:
Updated.